packages/markdown/package.json

```diff
@@ -40,6 +45,8 @@
     "eslint-plugin-react": "^7.14.3",
     "jest": "^24.8.0",
     "jsinspect": "^0.12.7",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "raw-loader": "^3.1.0",
+    "webpack": "^3.10.0"
```

Comment:
- [ ] I think (some of?) these can be moved/merged with the top-level Lerna repo's `devDependencies`?